ui/src/ScholarReader.tsx

```diff
@@ -52,6 +53,11 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
      * are called from outside ScholarReader.
      */
     this.state = {
+      userId: undefined,
```

Comment:
maybe set to null so it's explicit that we wanted it to not have a value and not oversight that it wasn't set?